src/erlcloud_ses.erl

```diff
@@ -132,6 +144,136 @@ delete_identity(Identity, Config) ->
         {error, Reason} -> {error, Reason}
     end.
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% Custom Verification Templates
+%% SES API:
+%% [https://docs.aws.amazon.com/ses/latest/APIReference/API_GetCustomVerificationEmailTemplate.html]
```

Comment:
One other thing: the rest of the module has documentation for each function, with a direct link to the AWS documentation for that API call and a succinct example. I think it would be best to use that same pattern here.

Reply:
If I move the functions in alphabetical order, then absolutely, each function should have a link to the actual API.